tests/commands/test_runner.py

```diff
@@ -321,6 +321,19 @@ def test_gevent_patch_all(self):
             out = subprocess.check_output(["ddtrace-run", "python", "tests/commands/ddtrace_run_gevent.py"])
             assert out.startswith(b"Test success")
 
+    def test_debug_mode(self):
+        with self.override_env(dict(DD_CALL_BASIC_CONFIG="true")):
+            p = subprocess.Popen(
+                ["ddtrace-run", "--debug", "python", "-c", "''"],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+
+            p.wait()
+            assert p.returncode == 0
+            assert p.stdout.read() == six.b("")
+            assert six.b("ddtrace.sampler") in p.stderr.read()
```

Comment:
We need to call basic config to assert `ddtrace.sampler` was logged.  A few tests in this PR need to be factored since they depend on the deprecated logic. I think it's best to refactor these tests in a future PR (like the tests which assert Span tags and metrics are set).

Reply:
Why do we do `ddtrace.sampler` here? Is it because it is one of the modules that logs at startup? Could we just check that there's any output to stderr instead?

Reply:
resolved offline: this was from the existing test; not going to change it here.